File: 2.0/java/contacts/updateContactList.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.contacts.model.request.UpdateContactListRequest;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api login", "api password");
        UpdateContactListRequest request = UpdateContactListRequest.create()
            .id(11646003L)
            .name("contact list updated")
            .build();
        client.contactListsApi().update(request);
    }
}

File: 2.0/csharp/contacts/updateContactList.cs
using CallfireApiClient;
using CallfireApiClient.Api.Contacts.Model.Request;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        var request = new UpdateContactListRequest
        {
            Id = 11646003,
            Name = "contact list updated"
        };
        client.ContactListsApi.Update(request);
    }
}
